### ISSUE TITLE
Strange error message on jack-in.

### ISSUE THREAD

**Issue by gmp26** [BetterThanTomorrow/calva#1275]
I can get a sticky error message on shadow-cljs jack-in. The error appears when you choose an app to connect to. Once it appears the only way to clear it is a complete VSCode restart. It may help to remove unused extensions - in my case removing scss-to-css helped - but again, not reliably. Here's the error:
https://gist.github.com/gmp26/406dc44251b50132d3b085259e077017

On MacOSX Big Sur 11.4, using Calva 2.0.209

**Comment by PEZ** [comment]
Thanks for reporting! Can you add to the issue description that this happens with shadow-cljs jack-in for you? (I happen to know, but it gives context for other readers of this issue). And also please give us some details about your operating system and version of VS Code.

**Comment by gmp26** [comment]
OK - I added shadow-cljs to the first sentence.  After removing scss-to-css again and restarting VSCode I have not had the issue reappear.

**Comment by PEZ** [comment]
I just released a new Calva which addresses other issues with shadow-cljs jack-in. I doubt it fixes this one, but who knows. Please let us know if you see the issue with `2.0.210`, and also if you don't see the issue for a few days or so.

**Comment by gmp26** [comment]
No problems throughout the day with scss-to-css uninstalled. I just reinstalled it to do some design rework and Bang! broken again, so it is looking more and more like there is interference between jack-in and scss-to-css. Haven't updated Calva yet.

**Comment by gmp26** [comment]
Right. Now using 2.0.210, and I've been round the loop quite a few times. 

I'm now positive that having scss-to-css installed causes jack-in to fail with this strange error. I do have other VSCode extensions installed - the complete list of other extensions that I use is 
* Calva
* Docker
* Git Graph
* GitLens
* HTML CSS Support
* JS-CSS-HTML Formatter.
No jack-in issue with all those installed. The issue is now always appearing when I add scss-to-css to the mix.

**Comment by bpringe** [comment]
Have you tested that the problem happens with only Calva and scss-to-css installed/enabled, and none of the others? If it still happens with only those two installed, that at least tells us it's just between scss-to-css and Calva, and doesn't somehow involve other extensions, though I doubt that is the case.

**Comment by gmp26** [comment]
@bpringe. Just done that - removed all extensions apart from Calva and jack-in works fine. Added scss-to-css and it fails. Removed scss-to-css and reinstalled the others and it worked again.

**Comment by PEZ** [comment]
This is a great find. Thanks for investigating it so thoroughly, @gmp26!

**Comment by gmp26** [comment]
@PEZ :) I needed it to work! My workaround is to use command line sass instead.

It's slightly worrying that extensions can interfere with each other in this way. I feel a need to read up on the VSCode plugin architecture to understand how extension security works. If they can get outside their sandbox I should not be so trusting of them.

**Comment by PEZ** [comment]
Is this still a problem, @gmp26?

**Comment by gmp26** [comment]
Not for me since I stopped using scss-to-css. But of course the problem may still remain for others
